internal/config/configuration.go

```diff
@@ -68,12 +69,44 @@ func Validate(config *Configuration) error {
 			return fmt.Errorf("entrypointUrl %s is not unique", spec.EntrypointURL)
 		}
 		uniquenessUrl[spec.EntrypointURL] = true
+
+		// Validate the storage
+		if err := registerAndvalidateStorage(spec); err != nil {
```

Comment:
Like `LoadSecurityFactory` , move this call on ` Load` method and call it `LoadStorage`